components/developer/gdb/gdb.p5m

```diff
@@ -68,6 +77,7 @@ file path=usr/share/gdb/syscalls/sparc64-solaris.xml variant.arch=sparc
 file path=usr/share/gdb/system-gdbinit/elinos.py pkg.tmp.autopyc=false
 file path=usr/share/gdb/system-gdbinit/wrs-linux.py pkg.tmp.autopyc=false
 file path=usr/share/info/annotate.info
+file path=usr/share/info/bfd.info
```

Comment:
... and this one with binutils:
```
# pkg search bfd.info
INDEX      ACTION VALUE                   PACKAGE
basename   file   usr/share/info/bfd.info pkg:/developer/gnu-binutils@2.39-2022.0.0.1
#
```

Reply:
Since this was also not included in the manifest before I removed it again.